plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/internal/ViewServlet.java

```diff
@@ -121,7 +122,9 @@ protected void service(final HttpServletRequest httpRequest, final HttpServletRe
       log.debug("Service completed");
     }
     catch (Exception e) {
-      log.warn("Service failure", e);
+      if (!(e instanceof AuthorizationException)) {
+        log.warn("Service failure", e);
+      }
```

Comment:
I needed this so I could see *real* problems! I'll undo it, it should probably be tackled more comprehensively elsewhere.